src/urllib3/__init__.py

```diff
@@ -94,6 +94,10 @@ def request(
     body: Optional[HTTPBody] = None,
     fields: Optional[_TYPE_FIELDS] = None,
     headers: Optional[Mapping[str, str]] = None,
+    preload_content: Optional[bool] = True,
+    redirect: Optional[bool] = True,
+    retries: Optional[Union[Retry, bool, int]] = None,
```

Comment:
I've left the `retries` to be `None`, it's by default `None` in the file [`connectionpool.py`](https://github.com/urllib3/urllib3/blob/main/src/urllib3/connectionpool.py#L629) at line#629 and afterwards it's converted to a proper `retries` object.